### ISSUE TITLE
Add Contexts if a Player is in the voice.yml channel/category

### ISSUE THREAD

**Issue by WillyWhitewool** [DiscordSRV/DiscordSRV#1298]
### Why?

Contexts for conditions are cool, and could allow people to make auto-updating ranks if a player is using proximity chat.

### What and How?

Contexts could be added for:
If a player is in category listed in voice.yml
If a player is muted or not on Discord (more specifically an user muting themselves, if possible to detect)

### Are there alternatives?

Placeholders would also work if these are more "secondary" since I believe contexts are usually for the most important data values to check.

### Checks

- [X] I have used the search at least once to check if my idea has already been suggested and perhaps already implemented.

**Comment by Comrad112** [comment]
So thinking about like 2 different placeholders
%discordsrv_connected% that shows if you're connected with voice
%discordsrv_indicator% which shows like a dot or red or green to see if they're close enough to talk

**Comment by WillyWhitewool** [comment]
Yeah, I was thinking something similar to that would be cool. If possible a context/placeholder for an user being muted in Discord would be interesting (you could connect a rank to it and it could show if someone's muted), not sure if Discord API can read that though. This is more the idea I had for the contexts/placeholders.
%discordsrv_connected% whether or not connected in voice.yml channel/category
%discordsrv_range% whether or not you are in any other players range (could also detect server mutes instead?)
%discordsrv_muted% whether or not you are muted

I have it set up on my server so if your account is linked you get a "✆" prefix. If these were added as contexts or placeholders there could definitely be some cool options for displaying information on how players are able to communicate: you could tell if someone was linked but not voice connected, and tell them to get connected. Or if someone was linked but muted, you could know they can hear you even if you can't hear them.
